screenpipe-app-tauri/components/video.tsx

```diff
@@ -120,7 +120,7 @@ export const VideoComponent = memo(function VideoComponent({
         </div>
       ) : (
         <video controls className="w-full rounded-md">
-          <source src={mediaSrc} type="video/mp4" />
+          <source src={mediaSrc} type="video/mp4; codecs=hvc1" />
```

Comment:
I tried a bunch of different things. `video-js`, `flv-h265.js`, but after a few hours of research and trying and error, I learned webkit (the browser thing Mac uses) has native support to h.265 through `hvc1` codec.